docs/source/es/_toctree.yml

```
@@ -96,6 +96,8 @@
     title: Mecanismos de atención
   - local: pad_truncation
     title: Relleno y truncamiento
+    local: peft
```

This should be in the "Tutorial" section.

```suggestion
   - local: peft
```